toolkit/tools/imager/imager.go

```diff
@@ -540,8 +593,13 @@ func buildImage(mountPointMap, mountPointToFsTypeMap, mountPointToMountArgsMap,
 	}
 
 	// Only configure the bootloader or read only partitions for actual disks, a rootfs does not need these
+	// TODO add extrat config to specify bootloader to use
```

Comment:
Is this still needed, or does the primary disk conver it?

Reply:
need to removed that comment. This would be resolved by the PrimaryDisk specified in systemconfig which tells us which disk to install the legacy bootloader on.